grc/utils/form_custom_validators.py

```diff
@@ -203,3 +203,40 @@ def file_length_check(form, field):
                 raise ValidationError(f'The selected file must be smaller than {max_size_in_mb}MB')
 
     return file_length_check
+
+
+def fileVirusScan(form, field):
+    if current_app.config['AV_API'] is None:
+        return
+    if (field.name not in request.files or request.files[field.name].filename == ''):
+        return
+
+    print('Scanning %s' % current_app.config['AV_API'], flush=True)
+    from pyclamd import ClamdNetworkSocket
+
+    uploaded = request.files[field.name]
+    uploaded.stream.seek(0)
+
+    url = current_app.config['AV_API']
+    url = url.replace('http://', '')
+    url = url.replace('https://', '')
+    if url.index(':'):
+        url = url[: url.index(':')]
+
+    cd = ClamdNetworkSocket(host=url, port=3310, timeout=None)
+    if not cd.ping():
+        print('Unable to communicate with virus scanner', flush=True)
+        return
+
+    results = cd.scan_stream(uploaded.stream.read())
+    print('results', flush=True)
+    print(results, flush=True)
```

Comment:
There's 5 `print` statements in this method.
Are they supposed to stay here for logging purposes, or were they just for testing?

Reply:
Removed these